app/src/main/java/com/orange/ods/app/ui/utilities/code/XmlCode.kt

```diff
@@ -42,6 +41,10 @@ object PredefinedXmlAttribute {
     class Id(value: String) : XmlAttribute("android:id", "@+id/$value")
     class LayoutHeight(matchParent: Boolean = false) : XmlAttribute("android:layout_height", getLayoutParamValue(matchParent))
     class LayoutWidth(matchParent: Boolean = false) : XmlAttribute("android:layout_width", getLayoutParamValue(matchParent))
+
+    open class AppAttr(name: String, value: String) : XmlAttribute("app:$name", value)
+    class DrawableAppAttr(name: String, drawableName: String) : AppAttr(name, "@drawable/$drawableName")
+    class DisabledAppAttr : AppAttr("enabled", "false")
```

Comment:
The naming of app and android attributes is not consistent: there is no suffix for android attribute but there is one for app attribute. However I don't know which one we should take as a reference.

Also, we could add an `AndroidAttr` class.